Makefile

```diff
@@ -3,16 +3,53 @@ src_dir = $(vendor_dir)/src
 lib_dir = $(vendor_dir)/lib
 include_dir = $(vendor_dir)/include
 
-vendor: create_out_dirs openh264
+make_args.x86_64-windows = \
+	CC=x86_64-w64-mingw32-gcc \
+	CXX=x86_64-w64-mingw32-g++ \
+	ARCH=x86_64 \
+	OS=mingw_nt
+make_args.x86_64-darwin = \
+	CC=o64-clang \
+	CXX=o64-clang++ \
```

Comment:
maybe `AR=llvm-ar` is required

Reply:
This is it. Cross compiling the example is fixed by this.

Reply:
Yep, it works!